### ISSUE TITLE
Every time I open the app it shows the Success! Dialog box. 

### ISSUE THREAD

**Issue by MixtecoBlue** [qklabs/qksms#60]
![screenshot_2015-05-19-20-31-26](https://cloud.githubusercontent.com/assets/12520926/7716842/225d40ae-fe66-11e4-9103-4d47245999fa.png)

**Comment by srcreigh** [comment]
Hmmm. Can you try the following things and tell me which one (if any) makes it stop?

1) Pressing "okay" instead
2) Tapping outside of the dialog
3) Actually exiting the app (open recents -> close the app)

**Comment by MixtecoBlue** [comment]
It's been doing it for two days. Of course now that I report the issue, it
doesn't do it. So I guess... Problem solved? Heh.

**Comment by srcreigh** [comment]
Hahaha :-) thanks for reporting the issue though!

We've been seeing this crop up. It's been getting better and better, but if it's happened to you then it's not perfect yet.

Please make another issue if you see it happening again!

**Comment by Blupi** [comment]
I'd this bug to be reopen as it occured to me when I installed the app with v2.0.1. on a nexus 4.
Switching to landscape mode also triggers it (back to vertical mode as well).
It definitively disappeared after I killed the app or rebooted the phone, I don't remember which.

A few weeks later, I migrated a nexus 5 to qksms v2.0.1 and the exact same behavior occured on first use.

I guess all new users experience it.
